tornado/test/httputil_test.py

```diff
@@ -261,6 +261,26 @@ def test_data_after_final_boundary(self):
 
 
 class HTTPHeadersTest(unittest.TestCase):
+
+    def test_parse_line_with_trailing_spaces(self):
+        headers = HTTPHeaders()
+
+        # Test header with multiple trailing spaces
+        headers.parse_line("Content-Length: 0   ")
+        self.assertEqual(headers.get('content-length'), '0')
+
+        # Test header with leading and trailing spaces
+        headers.parse_line(" Content-Length :  123  ")
```

Comment:
This test is failing not because of trailing whitespace in the value but because of the trailing whitespace in the key. (you'd see the value `123` in `headers.get('content-length ')`. Maybe whitespace in keys should raise an error (I think that's in the RFCs), but it's separate from the trailing whitespace in *values* (which is only a problem in the current code when continuation lines are used).